UefiCpuPkg/Test/Mock/Include/GoogleTest/Library/MockLocalApicLib.h

```diff
@@ -0,0 +1,32 @@
+/** @file MockLocalApicLib.h
+  Google Test mocks for LocalApicLib
+
+  Copyright (c) 2023, Intel Corporation. All rights reserved.

```

Comment:
Is this copyright correct?

Reply:
Changed Copyright info to Microsoft on MockLocalApicLib files